python/lsst/dax/imgserv/locateImage.py

```diff
@@ -450,9 +484,22 @@ def _cutoutBoxPixels(srcImage, xyCenter, width, height, log):
     imgSub = srcImage[pixULX:pixEndX, pixULY:pixEndY].clone()
     return imgSub
 
+
+def _getKeysFromList(flist, fields):
+    '''flist assumed to be dictionary
+    '''
+    vals = []
+    for f in fields:
+	for l in flist:
+	    if f == l:
+	        vals.append(flist[l])
+    return vals
```

Comment:
The lower case 'l' looks a log like a '1', and should be avoided as a variable name.

Would this work? 

```
for f in fields:
    vals.append(flist.get(f))
```